src/components/containers/Onboarding.js

```diff
@@ -11,9 +14,16 @@ import UserService from '../../services/UserService';
 import StorageService from '../../services/StorageService';
 
 const Intro = ({ handleContinue }) => {
+  const history = useHistory();
+
   const getUser = async () => {
     AuthService.signIn(function(user) {
-      console.log('Signed-in user:', user);
+      if (!user) {
+        StorageService.setLoggedIn(true);
+        history.push('/tasklist');
+      } else {
+        console.log('Signed-in user:', user);
```

Comment:
Just since we're getting close to launch, lets take out the console.logs especially when they have user info

Reply:
Sure yeah, it was in there before, so I just wasn't sure if I should have gotten rid of it yet!